### PR TITLE
Desktop version : make the project responsive for the desktop version

### PR DESCRIPTION
**Pull request by medaminedev66**
for the desktop version:
- Add new divs 
- Make all the sections responsive.

### CODE REVIEW

**Review by abruzy — CHANGES_REQUESTED** [review]
```markdown
Hi @medaminedev66 👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Good work
_1. You really did well with your commit message, bravo, keep that up._
_.2 Good job using flex and grid to layout your application, well done._

## Required Changes ♻️
_1. Kindly add a link to the live demo of your app to the readme file._
_2. Good job so far but your mobile design still needs some work to be done there. I examined your app using the browser dev tools and the layout were overlapping each other, please put in mind all various device witdth breakpoint such as 400px, 760px etc._
_3. Kindly add  more details on how to get a local copy of your app in your readme file._

Other than those changes, good job so far, please apply those changes and re-submit for review again.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by medaminedev66** [comment]
For the third required change: I added created the setup, the install and the usage of my project in the readme file, I don't know what I should add more.

File: index.html [hunk 1]
```diff
@@ -1,181 +1,277 @@
...
-                    <li class="btn-lng">html</li>
-                    <li class="btn-lng">Ruby</li>
-                    <li class="btn-lng">bootstrap</li>
-                </ul>
-                <button class="see-Project">See Project</button>
-            </div>
-        </section>
-        <section class="about-me">
-            <h2>About me</h2>
-            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```
Comment (Emmyn5600) [inline index.html:115]: - [ ] Optional, kindly, fix `about me` on the mobile version
abt1

**Review by Emmyn5600 — APPROVED** [review]
## STATUS: APPROVED :muscle::clap:

Hi, @medaminedev66 

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous code reviewer.

May I highlight the following:
- The repo looks professional:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have a descriptive PR:heavy_check_mark:
- Your live demo link is working properly:heavy_check_mark:
- You have a descriptive Readme file:heavy_check_mark:

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**